### PR TITLE
Fps first party

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds content explaining how to set up Fingerprint as a first-party entity on a client's site. Target audiences are Akamai SEs and customer SEs.

Abhishek - My biggest question in this is, "What do Macrometa folx do and where in the process?" We need to clarify that.

Also, we need high-level steps for the reverse/cloud proxy.

### CODE REVIEW

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+5. In the **Behaviors** section, set the **Origin Server Hostname** to be the PhotonIQ server.
+6. Enable the **True Client IP Header** option in the property configuration for the Fingerprint Forward Rule.
+7. Name the **True Client IP Header** option `True-Client-IP`.
+
+### Subdomain Setup
+
+This process requires adding DNS records to your site. Make sure you have the following permissions:
+
+- Necessary access permission to the DNS service to be able to create the DNS record under that subdomain
+- A valid SSL certificate for the domain. If using Let’sEncrypt, need to do the DNS challenge
```
Comment (joubert-macrometa) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]: I would not include Let's Encrypt stuff here.  Why speak of one of many different ways of doing it here.  And this does not even explain it.  It mentions one step of the many steps you have to do to get a lets encrypt certificate.

Comment (joubert-macrometa) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]:
```suggestion
- A valid TLS certificate for the subdomain.
```

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+If you are integrating Fingerprint outside of a cloud proxy, then you must create subdomain and alias it to the PhotonIQ server. This process requires adding a DNS records to your domain server:
+
+1. Create a subdomain off of your domain that points to the fps server.
+2. Set up the subdomain as an alias record, pointing it to the fps server.
+3. Get a valid SSL certificate for the domain. If using a product like Let’sEncrypt, then you need to do the DNS challenge.
```
Comment (joubert-macrometa) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]:
```suggestion
3. Get a valid TLS certificate for the domain.
```

Comment (joubert-macrometa) [reply docs/photoniq/fingerprinting/integrate-fingerprint.md]: We might want to mention what to do with the TLS certificate.

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+3. x
+
+#### Akamai Proxy Integration
+
+As an example, here are the steps to set up an Akamai proxy.
+
+If your plan supports then add property so that calls to the path `{your-website}/api/ds/*` are forwarded to `fps.photoniq.macrometa.io`.
+
+1. Log in to https://control.akamai.com/.
+2. In the **Properties** section, search for the property of the required URL. Add one if it doesn't exist already.
```
Comment (joubert-macrometa) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]: What is the "required" URL?

File: docs/photoniq/fingerprinting/integrate-fingerprint.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/fingerprinting/integrate-fingerprint.md]: So should I mention something about, "If your certificate is encrypted, then you might need to do a DNS challenge."?

File: docs/photoniq/fingerprinting/integrate-fingerprint.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/photoniq/fingerprinting/integrate-fingerprint.md]: Probably. So, what should we do with the TLS certificate?

File: docs/photoniq/fingerprinting/integrate-fingerprint.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/photoniq/fingerprinting/integrate-fingerprint.md]: Heck if I know. @lohaniabhishek ?

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Either strategy requires coordination with your Macrometa partner, because there needs to be an administrator on both sides in order to map the URLs for first-party integration.
+
+:::note
+With the release of Safari 16.4, cookie lifetime in Safari is reduced to 7 days when using the custom subdomain setup. We recommend using a cloud proxy integration to have cookies last up to one year in Safari.
+:::
+
+### Cloud Proxy Setup
+
+For first-party integration, you must ensure all traffic to the PhotonIQ server comes from your company's domain.  This is accomplished by redirecting all traffic to the server using a cloud proxy such as Akamai.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]: `This is accomplished by redirecting forwarding traffic to the server using a cloud proxy such as Akamai`

File: docs/photoniq/fingerprinting/integrate-fingerprint.md (hunk 3 above)
Comment (lohaniabhishek) [reply docs/photoniq/fingerprinting/integrate-fingerprint.md]: 'required url' will be the client's URL they are configuring for FPS to work on.

Comment (diana-macrometa) [reply docs/photoniq/fingerprinting/integrate-fingerprint.md]: How's this?

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+7. Name the **True Client IP Header** option `True-Client-IP`.
+
+### Subdomain Setup
+
+This process requires adding DNS records to your site. Make sure you have the following permissions:
+
+- Necessary access permission to the DNS service to be able to create the DNS record under that subdomain
+- A valid TLS certificate for the subdomain.
+
+If you are integrating Fingerprint outside of a cloud proxy, then you must create subdomain and alias it to the PhotonIQ server. This process requires adding a DNS records to your domain server:
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]: @lohaniabhishek re-read

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- Ad blockers will not block Fingerprint identification requests, because they are sent to the specified path or subdomain that belongs to the same site.
+
+## Integration Options
+
+There are two strategies for integrating Fingerprint as a first party:
+
+- (Recommended) Set up a cloud proxy using a tool such as Akamai, Cloudflare, or Nginx.
+- Set up a subdomain
+
+Either strategy requires coordination with your Macrometa partner, because there needs to be an administrator on both sides in order to map the URLs for first-party integration.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]: @lohaniabhishek what high-level task MM perform? DNS challenge

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+5. In the **Behaviors** section, set the **Origin Server Hostname** to be the PhotonIQ server.
+6. Enable the **True Client IP Header** option in the property configuration for the Fingerprint Forward Rule.
+7. Name the **True Client IP Header** option `True-Client-IP`.
+
+### Subdomain Setup
+
+This process requires adding DNS records to your site. Make sure you have the following permissions:
+
+- Necessary access permission to the DNS service to be able to create the DNS record under that subdomain
+- A valid TLS certificate for the subdomain.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]: will it be valid for other customers?
Better to be vague

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+- **Extended Visitor ID Lifespan and Ad Blocker Compatibility**: First-party cookies have a longer lifespan, which helps maintain visitor IDs over time. Additionally, since ad blockers typically allow connections to the same site's URLs, integrating the Fingerprint agent as first-party prevents it from being blocked, ensuring uninterrupted operation.
+- **Increased Security Against Fraudsters**: Making Fingerprint usage harder to detect for fraudsters adds an extra layer of security, safeguarding your data and the integrity of the identification process.
+
+## Integration Options
+
+There are two primary strategies for integrating Fingerprint as a first party, each requiring coordination with your Macrometa partner to ensure proper URL mapping for first-party integration.
+
+### Cloud Proxy Setup (Recommended)
+
+Use a cloud proxy service like Akamai, Cloudflare, or Nginx for first-party integration. This method involves redirecting traffic to the PhotonIQ server through a proxy hosted on your domain or subdomain, such as `metrics.yourwebsite.com`.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]:
```suggestion
Use a cloud proxy service like Akamai, Cloudflare, or Nginx for first-party integration. This method involves forwarding traffic to the PhotonIQ server through a proxy hosted on your domain or subdomain, such as `metrics.yourwebsite.com`.
```

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+Privacy is a crucial concern for many users. However, tools designed to protect user privacy, such as ad blockers and privacy-focused browsers, might interfere with Fingerprinting processes. These tools can block requests to Fingerprint servers, potentially preventing the identification of all visitors to your site.
+
+For more information about privacy and cookies, refer to [First- and Third-Party Cookies](first-third-party-cookie.md).
+
+## How the Fingerprint Agent Works on Your Website
+
+The Fingerprint agent, once installed on your website, performs several key actions:
+
+- **Downloads the Latest Device Intelligence Algorithm**: This ensures the most up-to-date method is used for identifying devices.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]:
```suggestion
- **Downloads the Latest Device Intelligence Script**: This ensures the most up-to-date method is used for identifying devices.
```

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+- **Collects and Sends Browser Attributes**: The agent sends collected data to the Fingerprint server and receives a unique visitor ID in return, facilitating accurate visitor identification.
+- **Makes Additional Requests**: These requests further enhance the accuracy of visitor identification.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]:
```suggestion
- **Makes Additional Requests**: These requests further enhance the accuracy of visitor identification by using advanced identification algorithm on the server side.
```

File: docs/photoniq/fingerprinting/integrate-fingerprint.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+## Challenges from Ad Blockers and Privacy Browsers
+
+Ad blockers (such as Adblock, uBlock Origin) and browsers with a focus on privacy (like Brave) actively block requests to Fingerprint servers. They do this by maintaining a list of Fingerprint domains and blocking browser requests that match these domains. Additionally, some VPN providers might block access to our domains at the DNS level, further complicating the identification process.
+
+To ensure the best performance and minimize disruptions, consider advising users on configuring their privacy tools to allow your site's Fingerprinting processes. This approach helps balance user privacy with the need for accurate visitor identification.
```
Comment (lohaniabhishek) [inline docs/photoniq/fingerprinting/integrate-fingerprint.md]:
```suggestion
To ensure the best performance and minimize disruptions, you should integrate PhotonIQ Fingerprinting as a first-party service in your website.
```

**Review by lohaniabhishek — APPROVED** [review]

**Pull request merged**